tests/api/test_predictions.py

```diff
@@ -3,10 +3,11 @@
 from httpx import Response
 
 from sync.api import predictions
+from sync.config import CONFIG
 from sync.models import Platform
 
 # auth route will only be called in the first test
-mock_router = respx.mock(base_url="https://*.synccomputing.com", assert_all_called=False)
+mock_router = respx.mock(base_url=CONFIG.api_url, assert_all_called=False)
```

Comment:
This started failing when running the tests pointed to my local API instance. Figured there was no harm in using this config value for the tests